Behat/Context/MJRContext.php

```diff
@@ -14,7 +14,7 @@
 use Modera\Component\SeleniumTools\Querying\By;
 use Facebook\WebDriver\WebDriverKeys;
 
-require_once 'PHPUnit/Framework/Assert/Functions.php';
+require_once __DIR__ . '/../../../../phpunit/phpunit/src/Framework/Assert/Functions.php';
```

Comment:
I think if PHPUnit is installed through composer to a project where SeleniumTools package is installed, then class loader generated by Composer should include proper path:namespace mapping for PHPUnit, so there's no need to hardcore the path.

Reply:
refactored. instead of using function now we use class PHPUnit_Framework_Assert inamespced functions